garage/np/algos/base.py

```diff
@@ -1,7 +1,26 @@
-class Algorithm:
-    pass
+from abc import ABC, abstractmethod
 
 
-class RLAlgorithm(Algorithm):
-    def train(self):
-        raise NotImplementedError
+class RLAlgorithm(ABC):
+    """Base class for all the algorithms.
+    """
+
+    @abstractmethod
```

Comment:
i think you can remove this from the interface, because it's not called from outside RLAlgorithm

Reply:
Remove @abstractmethod?